docs/contributing/_index.md

```diff
@@ -51,11 +51,13 @@ make
 with all the tools required. The source code is mounted from where you
 run `make` into the build container as a Docker volume.)
 
-To run the test suite:
+To run the unit tests suite:
```

Comment:
`make test` is so slow!

Running `go test ./...` is much faster for me.

Reply:
Also easier to run for Goland and attach debugger when I need to step through some code :)

Reply:
Good point, changed.